samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -77,11 +77,15 @@ static struct bt_mesh_sensor_value pres_mot_thres = {
 };
 static struct bt_mesh_sensor_value amb_light_level_ref = ILLUMINANCE_INIT_MILLIS(0);
 static float amb_light_level_gain = 1.0;
-/* Using a dummy ambient light value because we do not have a real ambient light sensor. */
+/* Using dummy sensor values to simulate having a real sensor. */
 static float dummy_ambient_light_value;
+static float dummy_people_count_value;
```

Comment:
It's a bit weird to use `float` for this since this is a count (it's 16 bit unsigned on the air). Suggest to use an integer type instead